bin/cli.js

```diff
@@ -1,8 +1,88 @@
 #!/usr/bin/env node
 'use strict';
 
-require('codemod-cli').runTransform(
-  __dirname,
-  process.argv[2] /* transform name */,
-  process.argv.slice(3) /* paths or globs */
-);
+function codemodCli() {
+  require('codemod-cli').runTransform(
```

Comment:
Let's *not* do this in the body of a function. The output from the generator was relatively reasonable as it's a top-level operation in that mode, but here it's *not*. Better to import and bind it:

```js
const codemodCli = require('codemod-cli');

function main() {
  codemodCli.runTransform(
    __dirname,
    process.argv[2] /* transform name */,
    process.argv.slice(3) /* paths or globs */
  );
}
```